src/bank01_player.asm

```diff
@@ -758,17 +766,16 @@ scrollRoom:
     ld   A, $00                                        ;; 01:453b $3e $00
     ld   [wScrollDirection], A                         ;; 01:453d $ea $41 $c3
     ret                                                ;; 01:4540 $c9
-    
-scrollRoomMove:
+.scrollRoomMove:
     ld   HL, wScrollPixelCounter
-    inc  C
-    swap C
+    inc  C ; $00 for first call, $01 for subsequent call
+    swap C ; $00 for first call, $10 for subsequent call
     xor  A, A
-    bit  1, E
+    bit  DIRECTIONB_WEST, E
     jr   NZ, .east
-    bit  2, E
+    bit  DIRECTIONB_NORTH, E
     jr   NZ, .south
-    bit  3, E
+    bit  DIRECTIONB_SOUTH, E
     jr   NZ, .north
 .west: ; by default if the others fell through
     ld   E, D
```

Comment:
What's going on here? I get that you're multiplying scroll speed by negative one, but there seems to be unnecessary register moves, especially considering it looks like `A == $00` at this point?

Reply:
If I did that I think I'd need to change a couple "jr"s to "jp"s. The optimization was intentional to save just enough bytes to keep it "jr"s. How about I just clarify what's going on with comments?

Reply:
I think I'd like simple comments on each of the vector transforms. Maybe something like:
`; D = $00, E = scroll amount times negative one`?

Reply:
Done